presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -897,11 +895,6 @@ private void testCreatePartitionedBucketedTableAsFewRows(Session session, HiveSt
 
         verifyPartitionedBucketedTableAsFewRows(storageFormat, tableName);
 
-        assertThatThrownBy(() -> assertUpdate(session, "INSERT INTO " + tableName + " VALUES ('a0', 'b0', 'c')", 1))
```

Comment:
can you do this insert and check it works?

Reply:
it's tested by a new test added in this PR.

Reply:
Exactly as @findepi has pointed out, I've moved it to a separate test, that should check only this one behaviour.